ui/scss/component/_button.scss

```diff
@@ -378,12 +378,52 @@ a.button--alt {
   }
 
   &:hover {
+    .button__label {
+      color: var(--color-link);
+    }
+  }
+}
+
+.button--comment-icons {
+  @extend .button--alt;
+  color: var(--color-text);
+  background-color: transparent;
+  margin-right: var(--spacing-m);
+  padding: 0 var(--spacing-xxs);
+  height: initial;
+  padding: 5px;
```

Comment:
There's a duplicate padding: line here. Also, whenever possible, we're trying to use vars for spacing, as above. Does one of those work?